src/www/pages/docs/get-started/production/index.njk.md

```diff
@@ -0,0 +1,88 @@
+{% extends "www/_layouts/layout-docs.njk" %}
+{% set pageTitle = "Production" %}
+{% set section = "Docs" %}
+{% from "wmnds/components/warning-text/_warning-text.njk" import wmndsWarningText %}
+
+{% block content %}
+
+{% markdown %}
+
+## About
+
+To start using our [styles](/styles/), [components](/components/) and [patterns](/patterns/) in your production project, you'll first need to include West Midlands Network Design System in your project.
+
+There are currently two ways of including the design system within your poject. You can either install it using node package manager (npm) or include the compiled files in your application.
+
+---
+
+### Option 1: include compiled files (recommended)
+
+Getting started is as simple as including West Midlands Network Design System's minified CSS file and adding <code class="wmnds-website-inline-code">.wmnds-html</code> to the html element in your service/application:
+
+#### Quick setup
+
+1. Add the below snippet of code to your page(s) within your project
+   <pre><code class="html wmnds-show-more-ignore" tabindex="0">
+     {%- filter forceescape %}
+     <!DOCTYPE html>
+     <html lang="en-gb" class="wmnds-html">
+       <head>
+         <!-- CSS for WMN Design System -->
+         <link rel="stylesheet" href="https://unpkg.com/wmn-design-system@$*version/build/css/wmnds.min.css" />
+       </head>
+       <body>
+         <!-- site content... -->
+       </body>
+     </html>
+     {%- endfilter %}
+     </code></pre>
+
+2. Add <a href='/styles/icons/#using-icons' title='Documentation about using icons' target='_blank'>West Midlands Network Design System icons</a> to your project.
```

Comment:
```suggestion
2. Add <a href='/styles/icons/#using-icons' title='Documentation about using icons' target='_blank' rel='noopener noreferrer'>West Midlands Network Design System icons</a> to your project.
```

Reply:
Not going to add the rel="noopener noreferrer" to this one as it is our website, so we want to promote SEO link growth.